hubblestack/extmods/modules/conf_publisher.py

```diff
@@ -41,38 +41,11 @@ def publish(report_directly_to_splunk=True, remove_dots=True, *args):
             if arg in  __opts__:
                 opts_to_log[arg] = __opts__[arg]
 
-    filtered_conf = _filter_config(opts_to_log, remove_dots=remove_dots)
+    filtered_conf = hubblestack.log.filter_logs(opts_to_log, remove_dots=remove_dots)
```

Comment:
Looks like we do an `emit_to_splunk` on line 47 -- I think we can eliminate this step and just add `remove_sensitive_logs=True` to line 47, right? Or am I missing something?

Reply:
Line 47 ( where we emit to splunk ) may or may NOT get executed depending upon the 'report_directly_to_splunk' flag.

Secondly, I have not exposed any parameter in "emit_to_splunk" function to pass the value of 'remove_dots' .

Because of these reasons, I have not deleted this step.

Reply:
You're right, of course. I was being silly. Thanks for catching that for me!